executor_test.go

```diff
@@ -27,3 +29,36 @@ func Test_ExecutorExecute(t *testing.T) {
 	e.stop()
 	wg.Wait()
 }
+
+func Test_ExecutorPanicHandling(t *testing.T) {
+	panicHandled := false
+	PanicHandler = func(jobName string, recoverData interface{}) {
+		fmt.Println("PanicHandler calld:")
+		fmt.Println("panic in " + jobName)
+		fmt.Println(recoverData)
+		panicHandled = true
```

Comment:
we run tests with the -race flag on, so it's complaining that you reference panicHandled in the main goroutine and then inside the func here. Could you add a mutex for it and lock/unlock here as well as line 63?